modules/ROOT/pages/apikit-for-soap-module-reference.adoc

```diff
@@ -0,0 +1,195 @@
+= APIkit for SOAP Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Configurations
+
+=== Config
+
+Configuration element for a *SOAP Router*
+
+==== Parameters
+
+[cols="1a,2a,3a,4a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | Connectors use this configuration name to reference its configuration. | | *x*
+| Soap Version | Enumeration, one of:
+
+* SOAP11
+ 
+* SOAP12 
+
+| SOAP version of the WSDL. |  | 
+| Address | String | Address of the web service. |  | 
+| MTOM Enabled | Boolean | Defines if the MTOM protocol manages the attachments. | `false` | 
+| Inbound Validation Enabled | Boolean | Defines if the MTOM protocol manages the attachments. | `false` | 
```

Comment:
The description of this item seems to be the same as the previous one, although the parameters are different. Please check if this is an error.

Reply:
OK! Thanks! 
Requesting assistance from SMEs.

Reply:
Great catch! It seems the table got broken somehow. 
Thanks!